src/utils/parsers/golang/index.ts

```diff
@@ -1,15 +1,13 @@
 import { BaseParser } from '../BaseParser'
 
-const authPattern = /\w*/
 const userCapturePattern = /(?:\w*|[\w\s.]*{[^})]*})/
 const variableNameCapturePattern = /([^,)]*)/
 const defaultValueCapturePattern = /(?:[^,)]*)/
 
 export class GolangParser extends BaseParser {
     identity = 'golang'
-    variableMethodPattern = /\.DevCycleApi\.Variable\(\s*/
+    variableMethodPattern = /\.Variable\(\s*/
```

Comment:
@jsalaber should this continue to support the older version of the SDK that includes `auth`?

Reply:
okay cool 👍